dags/bqetl_analytics_aggregations.py

```diff
@@ -118,6 +118,7 @@
         email=[
             "gkaberere@mozilla.com",
             "lvargas@mozilla.com",
+            "mozilla/kpi_table_reviewers",
```

Comment:
Will this cause any issues when sending emails since it is not technically an email address?

Reply:
I think it would. I added a change to [remove non-email owners ](https://github.com/mozilla/bigquery-etl/blob/08e997a2d80fd52434cba6584b4a6a64c1fa67fb/bigquery_etl/query_scheduling/task.py#L333-L341) when generating the email list for Airflow.